src/components/Rating.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const List = styled.li`
+  backgroundColor: '#e8f5e9',
+  margin: '1rem',
+  padding: '1rem',
+`;
+
+const Heading = styled.h3`
+  borderBottom: '1px solid rgba(0, 0, 0, 0.1)',
+  fontWeight: 300,
+  margin: 0,
+
+`;
```

Comment:
These won't work. Please check the [styled-components docs](https://www.styled-components.com/docs/basics), you write regular CSS between the backticks. So no camelCase, no single quotes, no commas.